### PR TITLE
Add Windows to run Unit tests

### PR DESCRIPTION
**Pull request by cooperlees**
Let's get some CI running on Windows.
- Remove .ptrconfig symlink
- Set encoding in setup.py to utf-8
- Make tests pass
- Add Windows runs to azure yml

Issue: #2

### CODE REVIEW

**Comment by cooperlees** [comment]
Can prob make this a lot cleaner but I've learnt a lot about Windows + Python. Would like to land this then do a clean up diff, then follow up with getting the integration runs happening on Windows too.
(Just trying to keep the diffs smaller)

File: ptr.py [hunk 1]
```diff
@@ -40,8 +48,8 @@ def _config_default() -> ConfigParser:
 def _config_read(cwd: str, conf_name: str = ".ptrconfig") -> Optional[ConfigParser]:
     """ Look from cwd to / for a "conf_name" file - If so read it in """
     cp = None
-    cwd_path = Path(cwd)  # type: Optional[Path]
-    root_path = Path("/")
+    cwd_path = Path(cwd)  # type: Path
+    root_path = Path("{}\\".format(cwd_path.drive)) if WINDOWS else Path("/")
```
Comment (zsol) [inline ptr.py]: Can you just use `cwd.anchor` here?

File: ptr_tests.py [hunk 2]
```diff
@@ -57,7 +56,7 @@ def return_specific_pid(*args: Any, **kwargs: Any) -> int:
 
 
 def return_specific_tmp(*args: Any, **kwargs: Any) -> str:
-    return "/tmp"
+    return "C:\\tmp" if ptr.WINDOWS else "/tmp"
```
Comment (zsol) [inline ptr_tests.py]: `tempfile.gettempdir` is your friend. I see you're using it down below, why not here?

File: ptr.py (hunk 1 above)
Comment (cooperlees) [reply ptr.py]: HOT - TIL - Thanks.

File: ptr_tests.py (hunk 2 above)
Comment (cooperlees) [reply ptr_tests.py]: I was trying to be more explicit but I guess it should always match on respective platforms.

File: ptr.py (hunk 1 above)
Comment (zsol) [reply ptr.py]: It didn't work? I thought line 52 could be replaced entirely by `root_path = cwd.anchor`

**Pull request merged**